.gitlab/kernel_matrix_testing/common.yml

```diff
@@ -332,8 +332,11 @@ notify_ebpf_complexity_changes:
             TEST_SET: no_usm
   allow_failure: true
   before_script:
-    - python3 -m pip install tabulate # Required for printing the tables
-    - python3 -m pip install -r tasks/libs/requirements-github.txt
+    # Python 3.12 changes default behavior how packages are installed.
+    # In particular, --break-system-packages command line option is
+    # required to use the old behavior or use a virtual env. https://github.com/actions/runner-images/issues/8615
+    - python3 -m pip install tabulate --break-system-packages # Required for printing the tables
```

Comment:
I think you can use the file `tasks/kernel_matrix_testing/requirements.txt` now

Reply:
+1 as the version isn't specifically set here in the job